tests/benchdnn/graph/graph.cpp

```diff
@@ -274,11 +279,13 @@ int make_input_tensors(std::vector<dnnl::graph::tensor> &input_ts,
 /// of the partition
 /// @param ops a list of op references of the partition
 /// @param outs a vector of logical tensors of partition outputs
+/// @param res a result structure to update output tensors size
 int make_output_tensors(std::vector<dnnl::graph::tensor> &output_ts,
         const graph::partition_mem_map_t &partition_mem_map,
         const graph::op_ref_list_t &ops,
         const std::vector<dnnl::graph::logical_tensor> &outs,
-        const std::vector<std::pair<size_t, size_t>> &inplace_ports) {
+        const std::vector<std::pair<size_t, size_t>> &inplace_ports,
+        res_t *res) {
```

Comment:
Please move the collection to this place: https://github.com/oneapi-src/oneDNN/blob/main/tests/benchdnn/graph/graph_memory.cpp#L46-L48